.github/workflows/ubuntu_clean_meson_build.yml

```
@@ -25,11 +25,23 @@ jobs:
       run: sudo apt-get update && sudo apt-get install -y gcc g++ pkg-config libopenblas-dev libiniparser-dev libjsoncpp-dev libcurl3-dev tensorflow2-lite-dev nnstreamer-dev libglib2.0-dev libgstreamer1.0-dev libgtest-dev ml-api-common-dev flatbuffers-compiler ml-inference-api-dev libunwind-dev
     - name: install additional packages for features
       run: sudo apt-get install -y python3-dev python3-numpy python3
+#    - name: add repo
+#      if: ${{matrix.os}} == 'ubuntu-22.04'
+#      run: sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+    - name: gcc version change
+      run: |
+        sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+        sudo apt-get install build-essential
+        sudo apt update
+        sudo apt install -y gcc-13
+        sudo apt install -y g++-13
+        sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-13 1000 
+        sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-13 1000
+        sudo update-alternatives --set gcc /usr/bin/gcc-13
```

let's set g++ with g++-13 as well.
```suggestion
        sudo update-alternatives --set gcc /usr/bin/gcc-13
        sudo update-alternatives --set g++ /usr/bin/g++-13
```